cli/azd/pkg/project/project_config.go

```diff
@@ -136,11 +136,28 @@ func ParseProjectConfig(yamlContent string, env *environment.Environment) (*Proj
 		if svc.Module == "" {
 			svc.Module = key
 		}
+
+		if svc.Language == "" || svc.Language == "csharp" || svc.Language == "fsharp" {
+			svc.Language = "dotnet"
+		}
 	}
 
 	return &projectFile, nil
 }
 
+func Initialize(ctx context.Context, p *ProjectConfig) error {
+	for _, svc := range p.Services {
+		frameworkService, err := svc.GetFrameworkService(ctx, &environment.Environment{})
```

Comment:
Consider accepting an `environment.Environment` parameter on the method.  We can likely cache the `env` passed in when loading the project initailly.